CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### Backward-incompatible changes [experimental]
 
 ### Bug fixes
+* Modified `skbio.stats.composition.ilr` and `skbio.stats.composition.ilr_inv` to suppress errors when inputing a 1D basis.  ([#1393](https://github.com/biocore/scikit-bio/issues/1393)).  Also fixing how the basis is handled in `skbio.stats.composition.ilr_inv`, the basis is now transformed back into the simplex space before checking to see if the basis is orthogonal.
```

Comment:
Instead of "suppress errors" how about "coerces rank to 2D" or something similar?

`Also ...` should be a second bullet with past tense. Examples would be good.